src/Monolog/LogProcessor.php

```diff
@@ -64,7 +64,7 @@ public function __invoke(array|LogRecord $record): array|LogRecord
                 'trace' => $exception->getTraceAsString(),
             ];
         }
-        $record['context'] = $context;
+        $record['extra']['context'] = $context;
```

Comment:
Mno ale kvuli radce `54` takle skonci cely context naduplikovany (zustane jako `context` + navic bude v `extra`).

Vidim tu tri moznosti:
* nechat `exception` v `context`
  * `JsonFormatter` zobrazi out-of-box `message` a `code` (to co jsou properties v exception)
  * do `extra` hodime `exceptionId` a `exceptionTrace`
* nebudeme tu upravovat `$record`, ale vytvorime novy pres `$record->with()` (https://github.com/Seldaek/monolog/blob/main/src/Monolog/LogRecord.php#L116)
  * bude tu muset byt `if` jestli je `$record instanceof LogRecord` nebo `array`
 * vyhodime handling `exception` s `LogPrcessor` a nasadimr `NormalizerFormatter` (https://github.com/Seldaek/monolog/blob/d3d495b742cacf9c0af87867bf87dab2fc591e68/src/Monolog/Formatter/NormalizerFormatter.php#L220)

Reply:
NormalizerFormatter tam nemáme schválně, je strašně náročnej na paměť

Reply:
> NormalizerFormatter tam nemáme schválně, je strašně náročnej na paměť

Tak to klidne muzem preformulovat na "nasadime neco jako `NormalizerFormatter` :slightly_smiling_face: Proste to prehodit processoru do formatteru a registrovat ho.

Reply:
Ale porad zbyvaji i 2 dalsi

Reply:
myslím, že to v tom processoru vůbec nemusí být